src/backend/joanie/payment/api.py

```diff
@@ -25,8 +26,9 @@ def webhook(request):
     payment_backend = get_payment_backend()
     try:
         payment_backend.handle_notification(request)
-    except exceptions.ParseNotificationFailed as error:
-        return Response(str(error), status=error.status_code)
+    except (exceptions.ParseNotificationFailed, ValidationError) as exception:
+        logger.error("Cannot handle payment notification", exc_info=exception)
+        return Response(str(exception), status=getattr(exception, "status_code", 500))
```

Comment:
What about capturing the exception in sentry ? You are returning the response with the status code 500, IMO all 500 should be catch by sentry.

I don't find where the ValidationError is raised. Can you show me please ?

Reply:
The ValidationError was the one fixed in the PR #443